src/ray/gcs/client-test.cc

```diff
@@ -0,0 +1,83 @@
+#include "gtest/gtest.h"
```

Comment:
Why `client-test.cc` not `client_test.cc`?

Reply:
That is how arrow names their tests, but you are right it is more consistent the other way so I changed it.